packages/web3-auth/src/types/Web3AuthConfig.ts

```diff
@@ -1,4 +1,9 @@
 export type DefaultSocialLoginConfig = {
-  defaultChainId: string
   backendUrl: string
 }
+
+export type SocialLoginDTO = {
+  chainId: string
+  whitelistUrls: { [P in string]: string }
```

Comment:
What does this type means? { [P in string]: string }

Reply:
The whitelistUrls expects an array of domain to signature which is passed to web3Auth on initialisation. So basically array of key string to value string.
Example 
```json
{
	"http://localhost:3000": "signature.....",
	"http://newdomain.com": "signature.....",
}
```